autoload/miniSnip.vim

```diff
@@ -62,7 +62,8 @@ function! s:directories() abort
 endfunction
 
 function! s:findSnippetFile() abort
-  let cword = matchstr(getline('.'), '\v\f+%' . col('.') . 'c')
+  let expandpattern = s:getVar("expandpattern")
```

Comment:
Is there any point into assigning it to variable instead of inlining?

Reply:
I am using this assignment to debug. But agree, could be removed.